src/test/scala/pureconfig/PureconfSuite.scala

```diff
@@ -373,12 +373,6 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with TryVal
     saveAndLoadIsIdentity(ConfWithQueueOfFoo(Queue(Foo(1), Foo(2))))
   }
 
-  case class ConfWithStackOfFoo(stack: collection.mutable.Stack[Foo])
```

Comment:
To preserve a test on a mutable data structure, we could switch to the mutable queue.
```
  case class ConfWithMutableQueueOfFoo(stack: collection.mutable.Queue[Foo])

  it should s"be able to save and load configurations containing mutable.Queue" in {
    saveAndLoadIsIdentity(ConfWithMutableQueueOfFoo(collection.mutable.Queue(Foo(1))))
  }
```